config/initializers/knock.rb

```diff
@@ -0,0 +1,59 @@
+Knock.setup do |config|
+
+  ## Expiration claim
+  ## ----------------
+  ##
+  ## How long before a token is expired. If nil is provided, token will
+  ## last forever.
+  ##
+  ## Default:
+  config.token_lifetime = 1.week
```

Comment:
The user token will expire automatically after 1 week. I don't have strong opinions on this, so happy to change it if people want this period to be longer/shorter.